api/Element.json

```diff
@@ -1183,10 +1183,10 @@
                 "version_added": null
               },
               "safari": {
-                "version_added": null
+                "version_added": "13.1"
```

Comment:
I can confirm my method arrives at the same answer:
https://trac.webkit.org/changeset/251043/webkit
https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/Configurations/Version.xcconfig?rev=251043